app/assets/stylesheets/ideas.scss

```diff
@@ -1,3 +1,23 @@
-// Place all the styles related to the Ideas controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+.ideas.listing {
+  margin-top:20px;
+}
+
+.favorite-icon {
+  &:hover {
+    cursor: pointer;
+
+    polygon {
+      stroke: #f3c967;
+    }
+  }
+}
+
+.new-idea-form {
+  background: #fff;
+  border:1px solid #e7e7e7;
+  border-radius: 4px;
+  margin: 20px auto;
```

Comment:
Properties should be ordered background, border, border-radius, float, margin, padding